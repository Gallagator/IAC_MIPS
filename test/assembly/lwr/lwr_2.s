    .text
	.align	2
	.globl	main
	.ent	main
main:  
	.set	noreorder
    lui   $3, 0x1234
    addiu $3, 0x5678
    sw    $3, 0($0)
    jr $0
    lwr   $2, 1($0) 
.end	main

