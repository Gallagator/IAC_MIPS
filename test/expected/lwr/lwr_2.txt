32'h56780000
